src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -146,6 +148,24 @@ const SettingsModalStackNavigator = () => (
             name="Settings_Profile"
             component={SettingsProfilePage}
         />
+        <SettingsModalStack.Screen
```

Comment:
Do we need two different screens (and two different routes) for this given the differences between them are pretty superficial and they route to the same component anyway?

Reply:
They re-use the same component but use `initialParams` to determine whether to show the phone/email copy

Reply:
Hmm, why not just determine the param type in `SettingsAddSecondaryLoginPage`?